plugins/destination/file/go.mod

```diff
@@ -53,7 +53,12 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 )
 
+require github.com/stretchr/testify v1.8.2
```

Comment:
move direct deps to 1 single place, indirect to another single place, go mod tidy?

Reply:
Good idea, fixed